float32 conf
---
geometry_msgs/Pose pose